### ISSUE TITLE
Is project still active?

### ISSUE THREAD

**Issue by peterchibunna** [jaz303/boxy#11]
Hello @jaz303, is this project still active?

**Comment by Qavai** [comment]
i think author is dead

**Comment by peterchibunna** [comment]
Nope, I think it's far from the truth. Author has recent activity on both github and twitter. I think he might have abandoned the project as other modals have become available, even Facebook modal that inspired this project, has moved on to another style.

**Comment by jaz303** [comment]
This project was built when before CSS had widespread support for rounded corners, transparent background colours, border radii, animations and `border-box`. Pretty much everything boxy does can now be trivially implemented with plain CSS and vanilla JS; I don't really see any need for it anymore.

**Comment by Qavai** [comment]
I still use your boxy in my projects, expand them, write additional code. it is quite handy, handy little library. in vain you threw it away. there is a lot of business to fix various bugs a little and that's it, let the project be afloat.
